installers/vagrant-dev/radiasoft-download.sh

```diff
@@ -171,18 +187,25 @@ EOF
     fi
     vagrant ssh <<EOF
 $(install_vars_export)
-curl $(install_depot_server)/index.sh | bash -s redhat-dev
+curl $(install_depot_server)/index.sh | \
+  vagrant_dev_ignore_git_dir_ownership=$(vagrant_dev_ignore_git_dir_ownership $os) \
```

Comment:
Typically we would name the variable at the place where it is used, e.g. you would set `bivio_home_env_ignore_git_dir_ownership`. This presumes that redhat-dev cascades the value, which it might need to(?).

Reply:
9b21050